pharmacy_retail/demo/sale_order_line.xml

```diff
@@ -0,0 +1,83 @@
+<odoo noupdate="1">
+    <record id="sale_order_line_10" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_20"/>
+        <field name="product_uom_qty">3.0</field>
+    </record>
+    <record id="sale_order_line_1" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_1"/>
+        <field name="product_id" ref="product_product_13"/>
+    </record>
+    <record id="sale_order_line_11" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_17"/>
+        <field name="product_uom_qty">4.0</field>
+    </record>
+    <record id="sale_order_line_12" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_15"/>
+        <field name="product_uom" ref="uom_uom_28"/>
+        <field name="product_uom_qty">3.0</field>
+    </record>
+    <record id="sale_order_line_13" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_16"/>
+    </record>
+    <record id="sale_order_line_14" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_18"/>
+        <field name="product_uom_qty">4.0</field>
+    </record>
+    <record id="sale_order_line_15" model="sale.order.line" context="{'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="order_id" ref="sale_order_3"/>
+        <field name="product_id" ref="product_product_26"/>
+        <field name="points_cost">1.0</field>
+        <field name="is_service" eval="True"/>
+    </record>
```

Comment:
You need to generate a coupon (demo) and call the function `sale_loyalty.sale_loyalty_coupon_wizard_action` to generate the discount